src/System.Diagnostics.DiagnosticSource/tests/DiagnosticSourceEventSourceBridgeTests.cs

```diff
@@ -15,7 +15,7 @@
 namespace System.Diagnostics.Tests
 {
     //Complex types are not supported on EventSource for .NET 4.5
-    [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework45)]
+    [SkipOnTargetFramework(TargetFrameworkMonikers.Net45 | TargetFrameworkMonikers.Net451 | TargetFrameworkMonikers.Net452)]
```

Comment:
@safern looks you still using the version netfx. if the test is running fine with the latest netfx, then remove the attribute. if still failing then keep the attribute but use TargetFrameworkMonikers.NetFx